ee/tabby-db/migrations/0014_password-reset.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE password_reset(
+    email VARCHAR(150) PRIMARY KEY NOT NULL,
+    code VARCHAR(36) NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT (DATETIME('now')),
+    FOREIGN KEY (email) REFERENCES users(email) ON DELETE CASCADE ON UPDATE CASCADE
```

Comment:
avoid using foreign key, consider use a cron job to delete staled records (like refresh_tokens table)

Reply:
Curious to know why?

Reply:
they're useful in certain scenario (e.g user / user order), but generally not too useful for cases like this (an ephemeral code), especially when combining with cascade deletion.

Reply:
Alright, do we want a single job which will handle cleaning up all expired resources, or a separate job for each of them?

Reply:
you might just add another function in https://github.com/TabbyML/tabby/blob/3db3d95eaedf7d347d035c1db87727cca46fe191/ee/tabby-webserver/src/cron/db.rs#L11, like `password_reset_job`